nativeApp/src/screens/admin_views/Time/TimePage/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC, useEffect } from 'react';
+import moment from 'moment';
+import { useSelector, useDispatch, shallowEqual } from 'react-redux';
+
+import TimeCard from '../../../../lib/ui/TimeCard';
+import Page from '../../../../lib/ui/Page';
+import useToggle from '../../../../lib/hooks/useToggle';
+import ConfirmationModal from '../../../../lib/ui/modals/ConfirmationModal';
+import CardSeparator from '../../../../lib/ui/CardSeparator';
+import { loadLogs, selectOrderedLogs } from '../../../../redux/entities/logs';
+import { loadVolunteers, selectVolunteers } from '../../../../redux/entities/volunteers';
+
+/*
+ * Types
+ */
+type Props = {
+}
+
+/*
+ * Styles
+ */
+
+/*
+ * Component
+ */
+const AdminTime: FC<Props> = () => {
+  const [visibleConfirmationModal, toggleDeleteVisibility] = useToggle(false);
+  const logs = useSelector(selectOrderedLogs, shallowEqual);
+  const volunteers = useSelector(selectVolunteers, shallowEqual);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(loadVolunteers());
+    dispatch(loadLogs());
+  }, []);
+
+  return (
+    <Page heading="Volunteers Time">
+      <ConfirmationModal
+        isVisible={visibleConfirmationModal}
+        onCancel={toggleDeleteVisibility}
+        onConfirm={toggleDeleteVisibility}
+        title="Delete"
+        text="Are you sure you want to delete this time?"
+      />
+      {/* TODO - separate logs into separator headers */}
+      <CardSeparator title="Today" />
+      {
+        logs.map((log) => (
+          <TimeCard
+            key={log.id}
+            id={log.id}
+            timeValues={[log.duration.hours || 0, log.duration.minutes || 0]}
+            labels={[log.project || 'General', log.activity]}
+            volunteer={volunteers[log.userId] ? volunteers[log.userId].name : 'Deleted User'}
```

Comment:
What about:
```ts
pathOr('Deleted User', [log.userId, 'name'], volunteers)
```

Reply:
I was going to avoid using ramda in this app as it has been notoriously annoying with TS. I was planning on writing small util functions or could trial something like lodash if you would prefer.

Reply:
I think a subset of ramda is annoying (curry, compose, pipe, evolve and related), but a lot of it is fine, and any problems with inference are usually indicative of type problems in application code.

Reply:
`curry` and `compose` is the reason I use ramda. There is no use to a function util library if you cannot use these.